### PR TITLE
Issue #SB-6729 feat: Course notifications bug & template fixes

### PR DESCRIPTION
**Pull request by iostream04**

### CODE REVIEW

File: cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql [hunk 1]
```diff
@@ -1,9 +1,12 @@
 ALTER TABLE sunbird.bulk_upload_process ADD storagedetails text;
```
Comment (bvinayakumar) [inline cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql]: Don't modify old CQL files.

File: cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
...nt-size: 14px; vertical-align: top;"> <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;"> You have been unenrolled from the course batch <strong>$batchName</strong>. Details of the batch are as given below: </p>#if($courseName )<p style=" float: left; margin: 0;">Course Name: $courseName </p>#end #if($batchName)<p style=" clear: both; float: left; margin: 0;">Batch Name: $batchName</p>#end</td></tr></tbody> </table> </td></tr></table> <p style="margin-bottom: 1px;"></p><p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 0;">Regards,</p><p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 0;">Team #if ($signature) <span> $signature</span> #end </p><p> Note: This is an automatic alert email. Replies to this mail box will not be monitored. If you are not the intended recipient of this message, or need to communicate with the team, write to $fromEmail. </p></td></tr><tr> <td> #if ($orgImageUrl) <p> <img src="$orgImageUrl" alt="logo" align="right" width="180" height="100"> </p>#end </td></tr></table> </div></td></tr></table></body></html>');
```
Comment (bvinayakumar) [inline cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql]: Add new statements in V1.37 CQL file as migration has been done in Sunbird Dev until V1.36 already.

File: cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql (hunk 1 above)
Comment (iostream04) [reply cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql]: Done

File: cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql (hunk 2 above)
Comment (iostream04) [reply cassandra-migration/src/main/resources/db/migration/cassandra/V1.33_cassandra.cql]: okay. Done created a new file

**Pull request merged**